src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/AddColumns.cs

```diff
@@ -78,6 +77,9 @@ public override bool CheckInvocation(TexlBinding binding, TexlNode[] args, DType
                 errors.EnsureError(DocumentErrorSeverity.Severe, args[0].Parent.CastList().Parent.CastCall(), TexlStrings.ErrBadArityOdd, count);
             }
 
+            var supportIndentifiers = binding.Features.HasFlag(Features.SupportIdentifiers);
+            string value = null;
```

Comment:
>value [](http://example.com/codeflow?start=19&length=5)

naming nit: `columnName`?

Reply:
Consider moving this declaration inside the `for` since we'd want to reset it each iteration.